db/Dockerfile

```diff
@@ -1,25 +1,25 @@
-FROM postgres:9.4
+FROM postgres:9.4-alpine
+
+ENV DEFAULT_TIMEZONE UTC
 
 # Install some packages to use WAL
-RUN apt-get update \
-    && apt-get install -y \
-      build-essential \
+RUN apk add --no-cache --virtual .build-deps \
+      bash \
+      build-base \
+      ca-certificates \
       curl \
-      daemontools \
+      libc6-compat \
       libffi-dev \
-      libssl-dev \
-      lzop \
+      linux-headers \
       pv \
       python-dev \
+      py-cffi \ 
+      py-cryptography \
+      tzdata \
     && curl --silent --show-error --retry 5 https://bootstrap.pypa.io/get-pip.py | python \
-    && pip install 'wal-e<1.0.0' \
-    && apt-get remove -y \
-      build-essential \
-      python-dev \
-    && apt-get autoremove -y \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin /tmp/* /var/tmp/*
-
+    && pip --no-cache-dir install 'wal-e<1.0.0' \
+    && rm -rf /var/cache/apk/* /tmp/* /var/tmp/* \
+    && apk del .build-deps
```

Comment:
here you removed `tzdata`. Isn't it being used for timezone info for postgres later when user sets timezone? And I think the same for `ca-certificates`, `pv`, `py-cffi` and `py-cryptography`. They are not build dependencies but runtime dependencies.

Reply:
error
[42897998.log](https://github.com/mattermost/mattermost-docker/files/1524356/42897998.log)

Reply:
fix, py-cryptography - without it does not work